src/devrules/cli_commands/prompters/typer_prompter.py

```
@@ -0,0 +1,288 @@
+                if not selection.strip():
+                    return []
+
+                indices = [int(x.strip()) for x in selection.split()]
+                selected = []
+                for idx in indices:
+                    if 1 <= idx <= len(options):
+                        selected.append(options[idx - 1])
+                    else:
+                        typer.secho(f"Invalid selection: {idx}", fg=typer.colors.YELLOW)
+
+                return selected if selected else None
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)